### ISSUE TITLE
Build digital story transporter

### ISSUE THREAD

**Issue by jennpb** [wellcometrust/wellcomecollection.org#671]
![image](https://cloud.githubusercontent.com/assets/1446139/23753334/fa0044d0-04d0-11e7-83a2-ab4722f15ebf.png)

**Comment by Norvard** [comment]
Updates and visual enhancements :)
Based on some other designs and discussions, had to make some updates to this. Examples below:
Also, since we have to adjust some of the colours based on accessibility when setting type over cream background. The blue we are using for the first new digital story, large text (numbers and dig story title) should be hex: #489199 (This is based on the Google Material accessibility tester)

Design examples below for desktop and mobile:
![screen shot 2017-05-08 at 11 41 49](https://cloud.githubusercontent.com/assets/23660297/25801029/be151468-33e3-11e7-8b65-e56f10211496.png)
![screen shot 2017-05-08 at 11 42 14](https://cloud.githubusercontent.com/assets/23660297/25801031/c31f0536-33e3-11e7-896d-6c0132761f9b.png)
